simulation/sub8_gazebo/nodes/gazebo_controller.py

```diff
@@ -56,7 +58,7 @@ def send_wrench(self, wrench):
             body_name=self.target,
             reference_frame='world',
             wrench=wrench,
-            duration=rospy.Duration(0.01)
+            duration=rospy.Duration(0.001)
```

Comment:
was the intentional?

Reply:
Not exactly but it didn't seem to change performance. Can change it back if desired.